dev/scf/values.yaml

```diff
@@ -1,2 +1 @@
----
-system_domain: cf-dev.io
+system_domain: scf.suse.dev
```

Comment:
Has Chris confirmed that this is set up ?

Reply:
This is a temporary value and is irrelevant at this point.

Reply:
Ok.